rskj-core/src/test/java/co/rsk/peg/federation/PowpegMigrationTest.java

```diff
@@ -62,6 +63,7 @@
 import static org.mockito.ArgumentMatchers.any;
 import static org.mockito.Mockito.*;
 
+@Disabled
```

Comment:
Is this now an official integration test? This explains why you decided to disable it, right?

Reply:
not exactly. A new task was created to address what to do with this test since it requires deeper analysis